modules/openapi-generator/src/main/resources/python-flask/requirements.mustache

```diff
@@ -1,4 +1,4 @@
-connexion >= 2.5.0; python_version>="3.6"
+connexion >= 2.6.0; python_version>="3.6"
```

Comment:
Why did you change this version?

Reply:
When using 2.5.0 I was getting an error about not being able to import `FileStorage` from `werkzeug `. The only way I could find to reliably fix this was to upgrade connexion as per [this answer](https://github.com/zalando/connexion/issues/1149#issuecomment-583848140) to [this issue](https://github.com/zalando/connexion/issues/1149)